File: src/main/java/org/zbus/rpc/RpcException.java
package org.zbus.rpc;

import org.zbus.mq.MqException;

public class RpcException extends MqException { 
	private static final long serialVersionUID = 8445590420018236422L;

	public RpcException(String message) {
		super(message); 
	}

	public RpcException() { 
	}

	public RpcException(String message, Throwable cause) {
		super(message, cause); 
	}

	public RpcException(Throwable cause) {
		super(cause); 
	}

}

File: api/csharp/RPC/RpcException.cs
using System;


namespace Zbus.RPC
{
    public class RpcException : Exception
    {
        public RpcException()
        {
        }

        public RpcException(string message)
            : base(message)
        {
        }
        public RpcException(string message, Exception inner)
            : base(message, inner)
        {
        }
    }
}
